index.php

```diff
@@ -17,10 +18,66 @@ function grabApi () {
         }
         $i ++;
         curl_setopt($curl, CURLOPT_URL, 'https://pokeapi.co/api/v2/pokemon/' . $i . '/');
-        $pokemon[] = curl_exec($curl);
+        $resp = curl_exec($curl);
+        $temp = json_decode($resp, TRUE);
+        $name = $temp['forms'][0]['name'];
+        $type1 = $temp['types'][0]['type']['name'];
+        $type2 = $temp['types'][1]['type']['name'];
+        $pokemon[] = [$name, $type1, $type2];
     }
     curl_close($curl);
     return $pokemon;
 }
 
-grabApi();
+/*
+ * Allows us to insert all of our data in our DB at once. By putting each into a () we can concatinate each and then insert into the db as one large value.
+ *
+ *$allPokemen - the array of all the pokemon (with the only data we needed).
```

Comment:
DOC BLOC  nedds to be written according to PSR2 standards. @param and @results missing.